test/software/amazon/ionschema/IonSchemaImportTest.kt

```diff
@@ -0,0 +1,43 @@
+package software.amazon.ionschema
+
+import org.junit.Assert.*
+import org.junit.Test
+import software.amazon.ion.system.IonSystemBuilder
+
+class IonSchemaImportTest {
+    private val ION = IonSystemBuilder.standard().build()
+    private val iss = IonSchemaSystemBuilder.standard()
+            .addAuthority(AuthorityFilesystem("data/test"))
+            .build()
+
+    @Test
+    fun getImportedType() {
+        assertNotNull(iss.loadSchema("schema/Customer.isl").getType("positive_int"))
+    }
+
+    @Test
+    fun getTypes() {
+        val types = iss
+                .loadSchema("schema/Customer.isl")
+                .getTypes()
+                .asSequence()
+                .associateBy({ it.name() })
+        assertTrue(types.contains("Customer"))
+        assertTrue(types.contains("positive_int"))
+    }
+
+    @Test
+    fun validateType() {
```

Comment:
This appears to be redundant with the tests defined in data/test/schema/Customer.isl.  The preference here is for the file-based tests (so they can be leveraged to verify the behavior of other Ion Schema impls).  Are you seeing additional value in having this test method?